build/extension_manifest.json

```diff
@@ -18,32 +18,47 @@
 	},
 	"content_scripts": [ {
 		"matches": [
-			"*://en.wikipedia.org/*",
 			"*://de.wikipedia.org/*",
-			"*://eu.wikipedia.org/*",
-			"*://tr.wikipedia.org/*",
+			"*://en.wikipedia.org/*",
 			"*://es.wikipedia.org/*",
+			"*://eu.wikipedia.org/*",
 			"*://fr.wikipedia.org/*",
+			"*://hu.wikipedia.org/*",
+			"*://id.wikipedia.org/*",
 			"*://it.wikipedia.org/*",
-			"*://pt.wikipedia.org/*"
+			"*://ja.wikipedia.org/*",
+			"*://nl.wikipedia.org/*",
+			"*://pl.wikipedia.org/*",
+			"*://pt.wikipedia.org/*",
+			"*://tr.wikipedia.org/*"
 		],
 		"exclude_globs": [
-			"*://en.wikipedia.org/wiki/Special:*",
+			"*://de.wikipedia.org/*title=Spezial:*",
 			"*://de.wikipedia.org/wiki/Spezial:*",
+			"*://en.wikipedia.org/*title=Special:*",
+			"*://en.wikipedia.org/wiki/Special:*",
+			"*://es.wikipedia.org/*title=Especial:*",
+			"*://es.wikipedia.org/wiki/Especial:*",
+			"*://eu.wikipedia.org/*title=Berezi:*",
 			"*://eu.wikipedia.org/wiki/Berezi:*",
-			"*://tr.wikipedia.org/wiki/%C3%96zel:*",
+			"*://fr.wikipedia.org/*title=Sp%C3%A9cial:*",
 			"*://fr.wikipedia.org/wiki/Sp%C3%A9cial:*",
+			"*://hu.wikipedia.org/*title=Speci%C3%A1lis:*",
+			"*://hu.wikipedia.org/wiki/Speci%C3%A1lis:*",
+			"*://id.wikipedia.org/*title=Istimewa:*",
+			"*://id.wikipedia.org/wiki/Istimewa:*",
+			"*://it.wikipedia.org/*title=Speciale:*",
 			"*://it.wikipedia.org/wiki/Speciale:*",
-			"*://es.wikipedia.org/wiki/Especial:*",
+			"*://ja.wikipedia.org/*title=%E7%89%B9%E5%88%A5::*",
```

Comment:
Looks good. Do we need two colons for Japanese? Curious since all the other languages only have a single colon.

Reply:
This is what code review is for! No, it shouldn't be there. Fixed :)